app/src/main/res/layout/activity_main.xml

```diff
@@ -431,10 +444,42 @@
                     android:text="@string/ar_navigation"
                     android:textColor="@android:color/white"
                     android:textSize="18sp"
-                    android:textStyle="bold" />
+                    android:textStyle="bold"/>
 
             </LinearLayout>
 
+            <!--<View-->
+                <!--android:layout_width="1dp"-->
+                <!--android:layout_height="match_parent"-->
+                <!--android:background="@drawable/menu_line_gradient_rotated_bg"/>-->
+
+            <!--<LinearLayout-->
```

Comment:
Should we hide it instead of commenting?

Reply:
agree, done